src/stubgen.py

```diff
@@ -564,7 +621,7 @@ def process_general(m: re.Match) -> str:
 
         return s
 
-    def apply_pattern(self, value: object, pattern: Pattern, match: re.Match) -> None:
+    def apply_pattern(self, value: object, pattern: Pattern, match: "re.Match[str]") -> None:
```

Comment:
From: https://stackoverflow.com/a/70847364

``for Python 3.8, use typing.Pattern[bytes]``